### PR TITLE
#196752 update user

### PR DESCRIPTION
**Pull request by snehalsabade**
PR: https://github.com/tekdi/sunbird-admin-ui/pull/11
Update Role-
Case-1
When - When user click on edit button of existing user showing popup
If- User is active
Then- Prepopulating user roles and User can select/unselect multiple roles
When - When user click on edit button of existing user
If- User is inactive
Then- User can not select roles/ popup can not be opened

Case- 2
When - When user click on edit button of new/create new user without refreshing page
Issue- When new user get created ,we are not able to prepopulate roles detail, because we get these roles under the organization array 

Implementation video :
https://www.awesomescreenshot.com/video/17649377?key=a445e39de0ef2c618a8a226a930a6c98

### CODE REVIEW

**Review by gouravmore — APPROVED** [review]

File: src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class AddEditUserComponent {
     this.messages = [];
     this.userService.addNewUser(payload).subscribe(response => {
       this.messages = [
-        { severity: 'success', summary: 'Success', detail: response.params.status }
+        { severity: 'success', summary: 'Roles added successfully', detail: response.params.status }
```
Comment (gouravmore) [inline src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.ts]: language const should use here

File: src/app/sb-admin/components/pages/sb-user/sbuser.component.ts [hunk 2]
```diff
@@ -97,10 +91,39 @@ export class SbUserComponent implements OnInit {
...
+          "userId": this.user.userId,
+          "organisationId": this.user.rootOrgId,
+          "roles": this.selectedUserRole
+        }
+      }
+      this.userService.saveUserRole(body).subscribe((response) => {
+        this.user.organisations[0].roles = this.selectedUserRole;
+        this.messages = [
+        ];
+        this.messageService.add({ severity: 'success', summary: 'Success', detail: response.params.status })
```
Comment (gouravmore) [inline src/app/sb-admin/components/pages/sb-user/sbuser.component.ts]: use language constants

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/app/sb-admin/components/pages/sb-user/sbuser.component.ts (hunk 2 above)
Comment (snehalsabade) [reply src/app/sb-admin/components/pages/sb-user/sbuser.component.ts]: removed summary

File: src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.ts (hunk 1 above)
Comment (snehalsabade) [reply src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.ts]: Removed summary

**Review by gouravmore — APPROVED** [review]

**Pull request merged**